app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-  has_many :events
```

Comment:
This relation also should stay but we will have problem that two different methods have the same name. This is not good but don't bother this too much now. I will explain you solution soon.